Rakefile

```diff
@@ -2,6 +2,7 @@ require 'rake/testtask'
 
 Rake::TestTask.new do |t|
   t.libs << 'test'
+  t.warning = false
```

Comment:
I don't think this is necessary. Only warnings are circular dependency warnings, which I'm getting rid of anyway, and one from webmock, which should exist if only to remind us to upgrade.